detect_secrets/core/audit.py

```diff
@@ -542,6 +548,19 @@ def _print_context(  # pragma: no cover
 
     if error_obj:
         raise error_obj
+    elif not exclude_remediation_note:
+        print(
+            '{}'.format(
+                colorize(
+                    (
+                        'A potential secret was detected in this code.'
+                        ' If so, it should be remediated.'
```

Comment:
what do you think about this instead? That way users know that they must remediate the secret, and mark this secret as valid

```suggestion
' If so, it should be remediated and marked as a valid secret for historical bookkeeping purposes.`
```

Reply:
Closing since I like Kyle's change better